Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp

```diff
@@ -346,7 +354,7 @@ AActor* ASimulationState::ServerSpawnEntity(const FROSSpawnEntityRequest& InROSS
     return newEntity;
 }
 
-AActor* ASimulationState::ServerSpawnEntity(const FROSSpawnEntityRequest& InRequest)
+AActor* ASimulationState::ServerSpawnEntity(const FROSSpawnEntityRequest& InRequest, const int32& InNetworkPlayerId)

```

Comment:
```suggestion
AActor* ASimulationState::ServerSpawnEntity(const FROSSpawnEntityRequest& InRequest, const int32 InNetworkPlayerId)
```
as POD, if passed as const, should be passed by value in general
https://www.sandordargo.com/blog/2020/11/25/when-use-const-4-parameters
https://www.faqcode4u.com/faq/476434/does-const-call-by-reference-improve-performance-when-applied-to-primitive-types